gammapy/irf/edisp/core.py

```diff
@@ -141,18 +153,11 @@ def to_edisp_kernel(self, offset, energy_true=None, energy=None):
 
         offset = Angle(offset)
 
-        if energy is None:
+        if energy_axis is None:
```

Comment:
Will this work correctly for the old behaviour? We have to support passing quantities for 1.3

Reply:
Apologies I removed a few lines when removed the warning.
Fixing now.